maintenance_mode/backends.py

```diff
@@ -89,3 +94,33 @@ def get_value(self):
     def set_value(self, value):
         value = self.from_bool_to_str_value(value)
         write_file(self._get_filepath(), value)
+
+
+class CacheBackend(AbstractStateBackend):
+    """
+    django-maintenance-mode backend which uses
+    [Django's cache framework](https://docs.djangoproject.com/en/dev/topics/cache/#django-s-cache-framework).
+    """
+
+    def get_value(self):
+        try:
+            value = cache.get("maintenance_mode", "0")
+        except Exception as e:
+            logger.warning(
```

Comment:
All other backends return `False` as default / fallback value in the `get_value` method, and this backend behaviour should consistent.

Why `"1"` is returned in case of error?

I personally think that this is also a dangerous behaviour because if there is any problem with the cache the whole project will result under maintenance.

So, I would return "0" by default and if you need to return "1" for project specific reasons, you can always extend this cache backend and handle the exception as you prefer.

Reply:
@fabiocaccamo, my thought processes was this ... in our case if the Cache backend was down or app is down. That's why we want our app in maintenance mode because if the cache is down we would get tons of 500 errors rolling in if it wasn't in maintenance mode.

Should we add a setting so you can have it return whatever you want it to if the Cache backend is down?

Reply:
@epicserve I understand your scenario, but I think it's a bit specific to your project, in your case you could:

In your `settings.py`

```python
MAINTENANCE_MODE_STATE_BACKEND = "yourapp.backends.MaintenanceModeCustomCacheBackend"
```

Then in your `yourapp.backends`

```python
from maintenance_mode.backends import CacheBackend


class MaintenanceModeCustomCacheBackend(CacheBackend):
    def get_value(self):
        try:
            value = super().get_value()
            return value
        except Exception:
            return True
     
```

Reply:
@fabiocaccamo,

Are you against adding something like `MAINTENANCE_MODE_CACHE_DOWN_STATE = True`? That would be nicer for us than having to subclass and add our own backend.

Reply:
@epicserve ok, let's add it. 

I would name it `MAINTENANCE_MODE_STATE_BACKEND_FALLBACK_VALUE` (`False` by default).
I prefer a more generic name, so it can be used also by other backends.

Please, could you also update the `README` with both: cache backend and this new setting?

Reply:
@fabiocaccamo, Done. ✅

Reply:
@epicserve could you rename the setting from `MAINTENANCE_MODE_CACHE_DOWN_STATE` to `MAINTENANCE_MODE_STATE_BACKEND_FALLBACK_VALUE`?